packages/aws-cdk-lib/aws-dynamodb/lib/shared.ts

```diff
@@ -527,6 +527,42 @@ export interface ITable extends IResource {
    *
    */
   metricSuccessfulRequestLatency(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for the successful request latency for GetItem operations
+   *
+   * @param props properties of a metric
+   */
+  metricSuccessfulRequestLatencyForGetItem?(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for the successful request latency for PutItem operations
+   *
+   * @param props properties of a metric
+   */
+  metricSuccessfulRequestLatencyForPutItem?(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for the successful request latency for Query operations
+   *
+   * @param props properties of a metric
+   */
+  metricSuccessfulRequestLatencyForQuery?(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for the successful request latency for Scan operations
+   *
+   * @param props properties of a metric
+   */
+  metricSuccessfulRequestLatencyForScan?(props?: cloudwatch.MetricOptions): cloudwatch.Metric;
+
+  /**
+   * Metric for the successful request latency for a specific operation
+   *
+   * @param operation The DynamoDB operation to get metrics for
+   * @param props properties of a metric
+   */
+  metricSuccessfulRequestLatencyForOperation?(operation: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric;
```

Comment:
Better yet, this could just be `metricSuccessfulRequestLatencyV2`, deprecating the old one. 

The lack of ability to pass in an operation could be seen as a serious defect that needs to be rectified.

Please tell me we at least had `TableName` in there already?